app/models/service_template.rb

```diff
@@ -129,7 +129,7 @@ def self.class_from_request_data(data)
   def update_catalog_item(options, auth_user = nil)
     config_info = validate_update_config_info(options)
     unless config_info
-      update_attributes!(options)
+      update_attributes!(options.except(:picture))
```

Comment:
Maybe we can simplify this by creating a `picture=` method that does something like this:

```def picture=(picture)
  super unless picture.kind_of?(Hash)
  
  super(create_picture(picture))
```

Reply:
The alternative is the calling code  not pass hashes but creates pictures and pass those to picture= so we dont' need to do this.